include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/ResultSummary.kt

```diff
@@ -19,6 +19,32 @@ data class ResultSummary(
     return json
   }
 
+  fun toHtml(): String {
+    return """
+        <h3>Summary</h3>
+        <table class="highlight">
+            <thead>
+            <tr>
+                <td>Total</td>
+                <td>Recorded</td>
+                <td>Added</td>
+                <td>Changed</td>
+                <td>Unchanged</td>
```

Comment:
These should probably also be `<th>` instead of `<td>`
```suggestion
                <th>Total</th>
                <th>Recorded</th>
                <th>Added</th>
                <th>Changed</th>
                <th>Unchanged</th>
```